### PR TITLE
SONAR-6457 Feed dependencies in batch report

### PR DESCRIPTION
**Pull request by henryju**

### CODE REVIEW

**Comment by SonarTech** [comment]
SonarQube analysis reported no new issues.

File: sonar-batch/src/main/java/org/sonar/batch/phases/DecoratorsExecutor.java [hunk 1]
```diff
@@ -43,21 +43,23 @@
...
+  private final SonarIndex index;
+  private final EventBus eventBus;
+  private final Project project;
+  private final CoverageExclusions coverageFilter;
+  private final MeasureCache measureCache;
+  private final MetricFinder metricFinder;
   private final DuplicationCache duplicationCache;
+  private final AnalysisMode analysisMode;
 
   public DecoratorsExecutor(BatchExtensionDictionnary batchExtDictionnary,
```
Comment (SonarTech) [inline sonar-batch/src/main/java/org/sonar/batch/phases/DecoratorsExecutor.java]: MAJORMethod has 9 parameters, which is greater than 7 authorized.[](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00107)

**Comment by SonarTech** [comment]
SonarQube analysis reported no new issues.

**Comment by julienlancelot** [comment]
LGTM

**Pull request merged**